testing/aws-testing-cdk/lib/aws-secrets-manager/SecretsManagerStack.ts

```diff
@@ -0,0 +1,26 @@
+import {SecretValue, Stack, StackProps} from 'aws-cdk-lib';
+import {Role} from 'aws-cdk-lib/aws-iam';
+import {Construct} from 'constructs';
+import {Secret} from 'aws-cdk-lib/aws-secretsmanager';
+
+export interface SecretsManagerStackProps extends StackProps {
+  readonly testingRole: Role;
+}
+
+export class SecretsManagerStack extends Stack {
+  readonly openSearchSinkSecrets : Secret;
+
+  constructor(scope: Construct, id: string, props: SecretsManagerStackProps) {
+    super(scope, id, props);
+
+    this.openSearchSinkSecrets = new Secret(this, 'OpenSearchSinkSecrets', {
+      secretName: 'opensearch-sink-basic-credentials',
+      secretObjectValue: {
+        // TODO: pass in secret values as args?
+        username: SecretValue.unsafePlainText('admin'),
```

Comment:
Maybe use a value which is less likely to be used elsewhere. For example: `data-prepper-admin`.

Reply:
Not sure how easy it is to reset the username/password with the opensearch docker image. Will leave it for future change.

Reply:
That makes sense.

We might want a different test which verifies secrets manager independently of OpenSearch. In that case, we'd want a more unique username/password.